commitlint.config.ts

```diff
@@ -174,7 +174,7 @@ module.exports = {
 
                 'too-many-spaces': ({raw}: {raw:any}) => {
                     let rawStr = convertAnyToString(raw, "raw");
-                    rawStr = removeCodeBlock(rawStr);
+                    rawStr = removeAllCodeBlocks(rawStr);
```

Comment:
@realmarv there's another rule that uses this technique so let's change it to use the new function (remember, in general, as a programmer, you should never duplicate code)

Reply:
Done